### PR TITLE
add amaninfowiki dump

### PR DESCRIPTION
**Pull request by Reception123**

### CODE REVIEW

**Comment by JohnFLewis** [comment]
Week is not a valid parameter. Even if it was, this would say 'Week two of every month' which is monthly and not what is described above.

**Comment by Reception123** [comment]
@JohnFLewis SPF sent me a link but I'm not exactly sure what parameter would fit here.

**Comment by NDKilla** [comment]
The current code would work but the question is do we really want to do this biweekly? also their aware that even running it bi-weekly, only the most up-to-date backup will be kept on static.mh.o?

**Comment by JohnFLewis** [comment]
"The current code would work"

@NDKilla false statement!

File: modules/mediawiki/manifests/dumps.pp [hunk 1]
```diff
@@ -4,11 +4,20 @@
...
         monthday => 1,
+     }
+    cron { 'Export amaninfowiki xml dump monthly':
+        ensure   => present,
+        command  => '/usr/bin/nice -n19 /usr/bin/php /srv/mediawiki/w/maintenance/dumpBackup.php --wiki amaninfowiki --logs --full > /mnt/mediawiki-static/dumps/amaninfowiki.xml',
+        user     => 'www-data',
+        minute   => '0',
+        hour     => '0',
+        month    => '*',
+        monthday => 15, 30,
```
Comment (JohnFLewis) [inline modules/mediawiki/manifests/dumps.pp]: nope!

monthday => [ '15', '30' ],

**Comment by NDKilla** [comment]
shit john this is why i freaking hate puppet

**Comment by NDKilla** [comment]
@JohnFLewis you can fix it in the freaking thingy >:(

**Comment by JohnFLewis** [comment]
But if I fix it, people don't learn why its wrong for the future! It's not like this has to be merged right now with urgency - otherwise I would have.

**Comment by Reception123** [comment]
So what should be done in this case?

**Comment by Reception123** [comment]
@Southparkfan @JohnFLewis @NDKilla User is waiting for updates on this.

**Pull request merged**